vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Commit `vendor` dir?

Reply:
Yes, as we don't have a build system for extension we need to commit the `vendor` directory so the dependencies are available, when creating a new ZIP file.